src/components/Inputs/CheckboxInput/CheckboxInput.tsx

```diff
@@ -0,0 +1,59 @@
+import style from './CheckboxInput.module.scss';
+import globalStyle from '@/GlobalClasses.module.scss';
+import Checkbox from '@/components/Buttons/Checkbox/Checkbox';
+import { RegisterCheckboxValue } from '@/services/types';
+import { useEffect, useState } from 'react';
+import { useFormContext } from 'react-hook-form';
+
+interface IChecboxInput {
+  title: string;
+  data: string[];
+}
+
+const CheckboxInput = ({ title, data }: IChecboxInput) => {
+  const { setValue } = useFormContext();
+
+  let initialRegisterValue;
+  switch (title) {
+    case 'Topping':
```

Comment:
**nit:** Cases should not be hardcoded as well here. Imagine that you have to add additional 100 titles, you would have to add new case for each one